src/client/firebase/config.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  FIREBASE_APP_API_KEY: '[YOUR CONFIG HERE]',
```

Comment:
So how would i add a specific key? Can we do that via environment variables in the frontend?

Reply:
We can but should we have that in the boilerplate?